### DISCUSSION TITLE
BASS Faulted with error code 46 and hitsounds delayed.

### DISCUSSION THREAD

**Discussion by tsbreuer** [ppy/osu#15235] [Q&A]
Tried booting osu! after getting a new audio setup.

However, upon launching there's a BASS error:
![imagen](https://user-images.githubusercontent.com/6902643/138369287-9f5fb019-c2b0-4f7d-b627-70d69b5adcb6.png)

And the hitsounds are NOTICEABLY delayed.

Any ideas?

**Comment by frenzibyte** [comment]
This is lacking information for action to take place. What's the platform you're running osu!lazer on? if Linux then it's likely something to do with your audio setup. Also please attach logs.

**Reply by tsbreuer** [reply]
After further research i ran into the conclusion it was because the device i was trying to use was already being taken control of. (Voicemeeter is rather possesive i guess)

As for the delay, it seems to be an inherent issue of my audio interface - scarlett 2i2, havent been able to find a way to lower the latency. I'm on windows
